operations/paymentrequests.md

```diff
@@ -0,0 +1,270 @@
+
+### Response
+
+```javascript
+{
+    "PaymentRequests": [
+        {
+            "Id": "bcc76295-4e47-4cf1-a7cb-afae00bd1c35",
+            "AccountId": "8466dfdd-0964-4002-8719-afa900d0f1ba",
+            "ReservationGroupId": null,
+            "State": "Pending",
+            "Amount": {
+                "Currency": "EUR",
+                "NetValue": 10,
+                "GrossValue": 10,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": 10,
+                            "TaxValue": 0
+                        }
+                    ]
+                }
+            },
+            "Type": "Payment",
+            "Reason": "PaymentCardDeclined",
+            "ExpirationUtc": "2023-02-23T23:00:00Z",
+            "Description": "Payment required.",
+            "Notes": null
+        }
+    ],
+    "Cursor": "bcc76295-4e47-4cf1-a7cb-afae00bd1c35"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `PaymentRequests` | array of [Payment requests](#payment-request) | required | The filtered payment requests. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest payment request returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older payment requests. |
+
+#### Payment request
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the payment request. |
+| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) to which the payment request was issued. |
+| `ReservationGroupId` | string | optional | Unique identifier of the [Reservation group](reservations#reservation-group). |
+| `State` | [Payment request state](#payment-request-state) | required | A payment request state. |
+| `Amount` | [Amount value](accountingitems.md#amount-value) | required | Amount of the payment request. |
+| `Type` | [Payment request type](#payment-request-type) | required | A payment request type. |
+| `Reason` | [Payment request reason](#payment-request-reason) | required | A payment request reason. |
+| `ExpirationUtc` | string | required | Date and time of the payment request's expiration in ISO 8601 format. |
+| `Description` | string | required | Description of the payment request. |
+| `Notes` | string | optional | Payment request's notes. |
+
+#### Payment request state
+
+* `Pending` - payment request is active and waiting for completion.
+* `Completed` - payment request was fulfilled.
+* `Canceled` - payment request was canceled.
+* `Expired` - payment request is past its expiration date and no longer active.

```

Comment:
Is this fixed sized? This says that it will never be extended, I think we should put the dots afterwards so we can add another options, same for the request type. @jonas-paul @VBuben